src/ui/save-html.js

```diff
@@ -63,38 +63,46 @@ const downloadLinks = [
   },
 ];
 
-function toDownloadLink(details) {
+/**
+ * @param {typeof downloadLinks[0]} details
+ */
+function toDownloadLink(details, conf) {
   const { id, href, fileName, title, type } = details;
+  const filename = conf.shortName
+    ? `${conf.shortName}-${ISODate.format(conf.publishDate)}`
```

Comment:
Is publishDate assured? Or should we not added it if missing?

Reply:
![image](https://user-images.githubusercontent.com/8426945/136013970-d006c988-7820-44a4-ae6b-adb0778c0b4b.png)
Should exist on each, but maybe good idea to add a failsafe.